src/lib/connectors/mssql/index.ts

```diff
@@ -0,0 +1,103 @@
+import { ConnectionPool, ConnectionPoolConfig } from "mssql";
+import { Connection } from "@/types";
+import { Connector } from "..";
+
+const systemDatabases = ["master", "tempdb", "model", "msdb"];
+
+const getMSSQLConnection = async (connection: Connection): Promise<ConnectionPool> => {
+  const connectionOptions: ConnectionPoolConfig = {
+    server: connection.host,
+    port: parseInt(connection.port),
+    user: connection.username,
+    password: connection.password,
+    database: connection.database,
+    options: {
+      encrypt: connection.ssl?.encrypt === true,
```

Comment:
maybe we should put `encrypt` into `Connection` instead of `Connection.ssl` if it's a special field for mssql.

Reply:
I'll do a refactor for this in the next PRs.